error: 'atomic' in namespace 'std' does not name a template type

Hello,

While trying to compile jacktrip on a raspberry pi 2 (Ubuntu 16.04.7 LTS / 4.4.0-1126-raspi2) under the dev branch (as I was running into the same error as reported in https://github.com/jacktrip/jacktrip/issues/246), I got some error reported below.

I managed to find a work around by adding the following to [UdpDataProtocol.h](https://github.com/jacktrip/jacktrip/blob/dev/src/UdpDataProtocol.h)
```cpp
#include "RingBuffer.h"
```
which, in turn, includes the encessary \<atomic\> on [line 44](https://github.com/jacktrip/jacktrip/blob/dev/src/RingBuffer.h#L44)

I'm not able to assess wether including \<atomic\> directly would be more relevant.

Error Log:
```
In file included from ../src/UdpDataProtocol.cpp:38:0:
../src/UdpDataProtocol.h:220:10: error: 'atomic' in namespace 'std' does not name a template type
     std::atomic<uint32_t> mTotCount;
          ^
../src/UdpDataProtocol.h:221:10: error: 'atomic' in namespace 'std' does not name a template type
     std::atomic<uint32_t> mLostCount;
          ^
../src/UdpDataProtocol.h:222:10: error: 'atomic' in namespace 'std' does not name a template type
     std::atomic<uint32_t> mOutOfOrderCount;
          ^
../src/UdpDataProtocol.h:223:10: error: 'atomic' in namespace 'std' does not name a template type
     std::atomic<uint32_t> mRevivedCount;
          ^
../src/UdpDataProtocol.cpp: In member function 'virtual void UdpDataProtocol::run()':
../src/UdpDataProtocol.cpp:602:9: error: 'mTotCount' was not declared in this scope
         mTotCount                = 0;
         ^
../src/UdpDataProtocol.cpp:603:9: error: 'mLostCount' was not declared in this scope
         mLostCount               = 0;
         ^
../src/UdpDataProtocol.cpp:604:9: error: 'mOutOfOrderCount' was not declared in this scope
         mOutOfOrderCount         = 0;
         ^
../src/UdpDataProtocol.cpp:607:9: error: 'mRevivedCount' was not declared in this scope
         mRevivedCount            = 0;
         ^
../src/UdpDataProtocol.cpp: In member function 'virtual void UdpDataProtocol::receivePacketRedundancy(int8_t*, int, int, uint16_t&, uint16_t&, uint16_t&)':
../src/UdpDataProtocol.cpp:744:15: error: 'mOutOfOrderCount' was not declared in this scope
             ++mOutOfOrderCount;
               ^
../src/UdpDataProtocol.cpp:748:17: error: 'mTotCount' was not declared in this scope
                 mTotCount     = 0;
                 ^
../src/UdpDataProtocol.cpp:752:13: error: 'mLostCount' was not declared in this scope
             mLostCount += lost;
             ^
../src/UdpDataProtocol.cpp:754:9: error: 'mTotCount' was not declared in this scope
         mTotCount += 1 + lost;
         ^
../src/UdpDataProtocol.cpp:773:5: error: 'mRevivedCount' was not declared in this scope
     mRevivedCount += redun_last_index;
     ^
../src/UdpDataProtocol.cpp: In member function 'virtual bool UdpDataProtocol::getStats(DataProtocol::PktStat*)':
../src/UdpDataProtocol.cpp:814:9: error: 'mLostCount' was not declared in this scope
         mLostCount       = 0;
         ^
../src/UdpDataProtocol.cpp:815:9: error: 'mOutOfOrderCount' was not declared in this scope
         mOutOfOrderCount = 0;
         ^
../src/UdpDataProtocol.cpp:816:9: error: 'mRevivedCount' was not declared in this scope
         mRevivedCount    = 0;
         ^
../src/UdpDataProtocol.cpp:818:24: error: 'mTotCount' was not declared in this scope
     stat->tot        = mTotCount;
                        ^
../src/UdpDataProtocol.cpp:819:24: error: 'mLostCount' was not declared in this scope
     stat->lost       = mLostCount;
                        ^
../src/UdpDataProtocol.cpp:820:24: error: 'mOutOfOrderCount' was not declared in this scope
     stat->outOfOrder = mOutOfOrderCount;
                        ^
../src/UdpDataProtocol.cpp:821:24: error: 'mRevivedCount' was not declared in this scope
     stat->revived    = mRevivedCount;
                        ^
```

--- a/src/UdpDataProtocol.h
+++ b/src/UdpDataProtocol.h
@@ -44,6 +44,7 @@
 #include <random>
 #include <stdexcept>
 #include <vector>
+#include <atomic>
 
 #include "DataProtocol.h"
 #include "jacktrip_globals.h"

